src/createServer.js

```diff
@@ -1,11 +1,280 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+
+  const app = express();
+
+  const users = [];
+
+  app.get('/users', express.json(), (req, res) => {
+    if (!users) {
+      res.statusCode = 200;
+
+      res.send([]);
+
+      return;
+    }
+
+    res.statusCode = 200;
+
+    res.send(users);
+  });
+
+  app.post('/users', express.json(), (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+      res.statusCode = 400;
+
+      res.send();
+
+      return;
+    }
+
+    const user = {
+      name,
+      id: users.length + 1,
+    };
+
+    users.push(user);
+
+    res.statusCode = 201;
+
+    res.send(user);
+  });
+
+  app.get('/users/:id', express.json(), (req, res) => {
+    const { id } = req.params;
+
+    const findUser = users.find((user) => user.id === Number(id));
+
+    if (!findUser) {
+      res.statusCode = 404;
+
+      res.send();
+
+      return;
+    }
+
+    if (Number.isNaN(Number(id))) {
+      res.statusCode = 400;
+
+      res.send();
+
+      return;
+    }
+
+    res.statusCode = 200;
+
+    res.send(findUser);
+  });
+
+  app.delete('/users/:id', express.json(), (req, res) => {
+    const { id } = req.params;
+
+    const findUser = users.find((user) => user.id === Number(id));
+
+    if (!findUser) {
+      res.statusCode = 404;
+
+      res.send();
+
+      return;
+    }
```

Comment:
Do you really need `send` here? Why you can't write just this way? 👇 
```suggestion
      return;
    }
```